src/helpers/driver_helpers.cpp

```diff
@@ -194,6 +194,27 @@ const naoqi_bridge_msgs::RobotInfo& getRobotInfo( const qi::SessionPtr& session
   return robot_info;
 }
 
+const std_srvs::Empty& startSpeechRecognition( const qi::SessionPtr& session)
+{
+  qi::AnyObject p_memory = session->service("ALSpeechRecognition");
+  std::cout << "speech recognition service loaded" << std::endl;
+  // I need to learn how to use dynamic reconfigure
+  p_memory.call<void>("setLanguage", "Japanese" );
+  std::list<std::string> vocab;
+  vocab.push_back("ペッパー");
+  vocab.push_back("ナオ");
+  p_memory.call<void>("setVocabulary", vocab, false);
+  //
```

Comment:
L. 201 - 207:
I think I have to delete these lines (I wrote these just for loading reconfigure test) 
and add lines for loading dynamic reconfigure.
Sorry for my trouble.